NewStarterProject/Controllers/AccessControlController.cs

```diff
@@ -0,0 +1,88 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.Identity.Client;
+using NewStarterProject.Dtos;
+using NewStarterProject.Model;
+using NewStarterProject.Services;
+
+namespace NewStarterProject.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class AccessControlController : ControllerBase
+    {
+
+        private readonly AccessControlService _accessControlService;
+
+        public AccessControlController(StarterProjectContext context)
+        {
+            _accessControlService = new AccessControlService(context);
```

Comment:
Can we try and see if we can use dependencu injection here instead of creating a new instance of the Access Control Service? 

https://learn.microsoft.com/en-us/aspnet/core/fundamentals/dependency-injection?view=aspnetcore-8.0